### ISSUE TITLE
not working with latest vite project

### ISSUE THREAD

**Issue by carl-jin** [serebrov/emoji-mart-vue#107]
got following error
``` shell script
[vite] Internal server error: Failed to resolve import "./Emoji". Does the file exist?
```

thats regular issue check this out [https://github.com/vitejs/vite/issues/178](https://github.com/vitejs/vite/issues/178)

just add `.vue` end of file path
``` javascript
export { default as Anchors } from './anchors.vue'
export { default as Category } from './category.vue'
export { default as Preview } from './preview.vue'
export { default as Search } from './search.vue'
export { default as Skins } from './skins.vue'
export { default as Emoji } from './Emoji.vue'
export { default as Picker } from './Picker.vue'
``

**Comment by carl-jin** [comment]
[https://github.com/carl-jin/emoji-mart-vue/commit/117042cb6bb627c8ad99fd0f2113f68bcb648ed4](https://github.com/carl-jin/emoji-mart-vue/commit/117042cb6bb627c8ad99fd0f2113f68bcb648ed4)

check this commit above

**Comment by serebrov** [comment]
@carl-jin I've just published version 9.1.0 (also includes emoji v13). Here is the change - https://github.com/serebrov/emoji-mart-vue/pull/110/files, I added ".vue" extensions as you suggested.

Could you help testing it?

**Comment by carl-jin** [comment]
thats my pleasure, let me check it out

**Comment by carl-jin** [comment]
got an error, 
``` shell script
SyntaxError: The requested module '/node_modules/emoji-mart-vue-fast/src/utils/data.js' does not provide an export named 'buildSearch'
```
change following code for fast fix it [https://github.com/serebrov/emoji-mart-vue/blob/976448176ae41b911116172d41509ce190dd1401/src/utils/data.js#L122](https://github.com/serebrov/emoji-mart-vue/blob/976448176ae41b911116172d41509ce190dd1401/src/utils/data.js#L122)
``` javascript
export { buildSearch, compress, uncompress }
// module.exports = { buildSearch, compress, uncompress }
```

**Comment by carl-jin** [comment]
> thats my pleasure, let me check it out

**Comment by carl-jin** [comment]
> @carl-jin I've just published version 9.1.0 (also includes emoji v13). Here is the change - https://github.com/serebrov/emoji-mart-vue/pull/110/files, I added ".vue" extensions as you suggested.
> 
> Could you help testing it?

**Comment by serebrov** [comment]
@carl-jin I published `9.1.2` with the export fix.
